distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -2335,7 +2335,7 @@ type Column
     to_js_object self =
         name = self.java_column.getName
         storage = self.java_column.getStorage
-        storage_proxy = Array_Proxy.new storage.size i-> storage.getItem i
+        storage_proxy = Array_Proxy.new storage.size i-> storage.getItemBoxed i
```

Comment:
`getItem` isn't defined in `Storage` class, but only in individual subclasses. I don't want to (I tried) import all the individual subclasses of `Storage`, so I am changing this code to call the `Storage.getItemBoxed` method. The call is reflective so boxing (in the interpreter; before compilation) is inevitable anyway.

Benchmarks shall not be affected: check [this run](https://github.com/enso-org/enso/actions/runs/11495022958).

Reply:
> reflective so boxing (in the interpreter; before compilation) is inevitable anyway.

In the interpreter yes, but won't it make it less likely to get nicely inlined once compiled?

(Just asking out of curiosity)

Reply:
[Escape analysis](https://ssw.jku.at/Teaching/PhDTheses/Stadler/Thesis_Stadler_14.pdf) effectively removes all boxing in compiled code.

Reply:
Executed:
```bash
enso/tools/performance/engine-benchmarks$ ./bench_download.py -s stdlib -b wip/jtulach/GeoTestsInNativeImage develop
```
and opened `enso/tools/performance/engine-benchmarks/generated_site/stdlib-benchs.html` in browser. I **don't see any problem** that should stop us from moving forward.